src/java/org/apache/cassandra/cql3/terms/Lists.java

```diff
@@ -300,7 +300,7 @@ public Setter(ColumnMetadata column, Term t)
         @Override
         public boolean requiresTimestamp()
         {
-            return true;
+            return column.type.isMultiCell();
```

Comment:
bug found in fuzz testing, frozen lists were requiring timestamp which made it hit this block

```
public void add(Operation operation, TableMetadata tableMetadata)
    {
        if (isForTxn && (operation.requiresRead() || operation.requiresTimestamp()))
        {
            add(operation.column, ReferenceOperation.create(operation, tableMetadata));
            return;
        }
```

this is a immutable bb so can avoid converting to a reference (i think it fails as a reference but forgot)